Harubee-iOS/Sources/Presentation/Onboarding/NavigationHeaderView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NavigationHeaderView.swift
+//  Harubee-iOS
+//
+//  Created by Seo-Jooyoung on 11/2/24.
+//  Copyright © 2024 namudiEtc. All rights reserved.
+//
+
+import SwiftUI
+
+enum PageNumber {
+  case first, second, third
+  
+  var pageNumber: Int {
+    switch self {
+    case .first:
+      return 1
+    case .second:
+      return 2
+    case .third:
+      return 3
+    }
+  }
+}
```

Comment:
다음과 같은 네이밍은 어떨까요?
일단 온비딩에서만 사용되는 열거형이고,  기존 pageNumber.pageNumber는 이름이 중복되어 사용되었기 때문에 OnboardingPage라는 이름으로 지어봤습니다.

```suggestion

enum OnboardingPage {
  case first, second, third
  
  var numberr: Int {
    switch self {
    case .first:
      return 1
    case .second:
      return 2
    case .third:
      return 3
    }
  }
}
```

Reply:
좋은 것 같습니다 ! 수정해서 적용했습니다 ~